hooks/userhooks.php

```diff
@@ -10,7 +10,10 @@
 
 namespace OCA\AgreeDisclaimer\Hooks;
 
-use OCA\AgreeDisclaimer\AppInfo\Application;
+use OCP\IUserManager;
+use OCP\IL10N;
+
+// hint: private code usage is bad!
 use OC\User\LoginException;
```

Comment:
Depending on OC\ namespace classes will break your code in future ownCloud versions, only use OCP. Don't know if this is possible in this case however